ParadigApp/Assets/Scripts/Login.cs

```diff
@@ -1,31 +1,37 @@
 ﻿using UnityEngine;
 using System.Collections;
 
-public class Login : MonoBehaviour {
+public class Login : MonoBehaviour 
+{
 
 
 	string userName;
 	string password;
 
-	public void setUserName(string pUserName){
+	public void setUserName(string pUserName)
```

Comment:
We don't use getters/setters in c#... use properties instead 
https://msdn.microsoft.com/en-us/library/x9fsa0sw.aspx